### PR TITLE
Add swedish in default pipelines

### PR DESCRIPTION
**Pull request by ManyTheFish**
# Summary
## Fix Swedish support

In Swedish the characters `å`/`ä`/`ö` are completely different than `a` or `o`  and should not be normalized as the same character.
because the Swedish specialized pipeline was not activated by default, these characters were normalized even with the settings:
```json
{
  "localizedAttributes": [ { "locales": ["swe"], "attributePatterns": ["*"] } ]
}
```

## Update Charabia adding German support

German segmentation will now be activated using the setting:
```json
{
  "localizedAttributes": [ { "locales": ["deu"], "attributePatterns": ["*"] } ]
}
```

# TODO

- [x] Activate Swedish Pipeline
- [x] Add a test to avoid future regressions
- [x] Update Charabia

### CODE REVIEW

**Comment by dureuill** [comment]
🤔 

https://bench.meilisearch.dev//view_span?span=indexing::extract::extract_docid_word_positions&workload_name=hackernews.ndjson_1M&target=bbaee3dbc63640984051e3eb37e7fc0e57dd873e&baseline=ff523a2357767f5c64296524a01b0ec35bdd33ca&baseline_branch=main&target_branch=add-swedish-in-default-pipelines

**Comment by ManyTheFish** [comment]
We still have a regression even if there is no more language detection 🤔

**Comment by effata** [comment]
@ManyTheFish Please note that Swedish has three of these characters: `å`, `ä` & `ö`, and they all seem to have the wrong behaviour atm.

**Comment by ManyTheFish** [comment]
@effata, Yes no worries they are all 3 handled, the normalizer was completely deactivated

**Comment by ManyTheFish** [comment]
@dureuill I deactivated the compilation flag in the last commit but nothing changes, it's even worse.. 🤔 so weird

**Comment by dureuill** [comment]
Hey @ManyTheFish :wave:

According to #4947 the regression doesn't seem linked with the current PR. We will investigate this during pre-release.

Please re-enable Swedish and German.

Nitpick: Please fix the order of locales in the error message

File: meilisearch-types/src/locales.rs [hunk 1]
```diff
@@ -79,7 +83,7 @@ macro_rules! make_locale {
 
         impl std::fmt::Display for LocaleFormatError {
             fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-                let valid_locales = [$($iso_639_1_str),+,$($iso_639_3_str),+].join(", ");
+                let valid_locales = [$($iso_639_1_str),+,$($iso_639_3_str),+,"cmn"].join(", ");
```
Comment (dureuill) [inline meilisearch-types/src/locales.rs:86]:
```suggestion
                let mut valid_locales = [$($iso_639_1_str),+,$($iso_639_3_str),+,"cmn"];.join(", ");
                valid_locales.sort();
                let valid_locales = valid_locales.join(", ");
```

**Review by dureuill — APPROVED** [review]

**Comment by dureuill** [comment]
bors merge

**Pull request merged**